proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -80,11 +100,135 @@ message Event {
+    PayloadOrganizationMemberRoleChanged organization_member_role_changed = 9;
+    PayloadOrganizationMemberRemoved organization_member_removed = 10;
+    PayloadRepositoryCreated repository_created = 11;
+    PayloadRepositoryDeleted repository_deleted = 12;
+    PayloadRepositoryPushed repository_pushed = 13;
+    PayloadRepositoryMemberAdded repository_member_added = 14;
+    PayloadRepositoryMemberRoleChanged repository_member_role_changed = 15;
+    PayloadRepositoryMemberRemoved repository_member_removed = 16;
+    PayloadPluginCreated plugin_created = 17;
+    PayloadPluginDeleted plugin_deleted = 18;
+    PayloadPluginPushed plugin_pushed = 19;
+    PayloadUserRegistered user_registered = 20;
+    PayloadUserDeactivated user_deactivated = 21;
+    PayloadUserDeleted user_deleted = 22;
+    PayloadUserLoggedIn user_logged_in = 23;
+    PayloadUserLoggedOut user_logged_out = 24;
   }
 }
 
-// PayloadOrganizationCreated is the payload for an ORGANIZATION_CREATED event.
 message PayloadOrganizationCreated {
-  // created_time is when the organization was created.
-  google.protobuf.Timestamp created_time = 1;
+  // create_time is when the organization was created.
+  google.protobuf.Timestamp create_time = 1;
 }
+
+message PayloadOrganizationDeleted {}
+
+message PayloadOrganizationMemberAdded {
+  // member_id is the id of the member added to the organization.
+  string member_id = 1;
+  // member_name is the name of the member added to the organization.
+  string member_name = 2;
+  // new_role is the role granted to the member added to the organization.
+  buf.alpha.registry.v1alpha1.OrganizationRole new_role = 3;
+}
+
+message PayloadOrganizationMemberRoleChanged {
+  // member_id is the id of the member whose role was changed.
+  string member_id = 1;
+  // member_name is the name of the member whose role was changed.
+  string member_name = 2;
+  // old_role is the old role of the member whose role was changed.
+  buf.alpha.registry.v1alpha1.OrganizationRole old_role = 3;
+  // new_role is the new role of the member whose role was changed.
+  buf.alpha.registry.v1alpha1.OrganizationRole new_role = 4;
+}
+
+message PayloadOrganizationMemberRemoved {
+  // member_id is the id of the member removed from the organization.
+  string member_id = 1;
+  // member_name is the name of the member removed from the organization.
+  string member_name = 2;
+  // old_role is the old role of the member removed from the organization.
+  buf.alpha.registry.v1alpha1.OrganizationRole old_role = 3;
+}
+
+message PayloadRepositoryCreated {
+  // create_time is when the repository was created.
+  google.protobuf.Timestamp create_time = 1;
+}
+
+message PayloadRepositoryDeleted {}
+
+message PayloadRepositoryPushed {
+  // owner_id is the id of the owner of the repository.
+  string owner_id = 1;
+  // owner_name is the name of the owner of the repository.
+  string owner_name = 2;
+  // commit is the name of the created commit.
+  string commit = 3;
```

Comment:
`commit_name` is already in the resource. You're missing repo name and repo id.

Reply:
I think I was thinking that the repository is the thing that the action "happens to", so was going with the repository as the resource. But we can change that to be the commit?

Reply:
Yeah, that was proposed in [the list](https://www.notion.so/bufbuild/Audited-Events-List-db1e266e05ce4d0db48638ec6091b9b2), use the commit as the resource. Why do you think it makes more sense to have the repo as the resource?

Reply:
ah, sorry, I didn't look closely enough at that. I wasn't viewing commits as a "resource" - I guess they are, though. I don't have a strong feeling about it; we're capturing essentially the same information either way. I'll change it.